pages/osx/awk.md

```diff
@@ -0,0 +1,24 @@
+# awk
+
+> Find and Replace text within file(s).
+> More information: <https://ss64.com/osx/awk.html>.
+
+- Define the input field separator = regular expression fs:
+
+`awk [ -F fs ]`
+
+- Assign values before prog is executed:
+
+`awk [ -v var=value ]`
+
+- Specify a pattern to match within progfile:
+
+`awk [ 'prog' | -f progfile ]`
+
+- Set the maximum size of the input record (MaxRows):
+
+`awk -mr`
```

Comment:
```suggestion
- Set the maximum size of the input record:

`awk -mr {{1..infinity}}`
```

Optional fix: description can be simplified.
Optional fix: `-mr` possibly accepts one non-integer number.